### ISSUE TITLE
Mandatory type bind field in submission form causes trouble

### ISSUE THREAD

**Issue by olli-gold** [4Science/DSpace#18]
In our publication form we have some type bound fields, which are mandatory for the bound type (but of course not for other types). The type bound fields are only displayed, if the type matches the bind definition - this is correct. But there is a bug, if the type bind fields are mandatory. If I publish anything with a different type, its impossible to proceed the publication. There are no error messages, because the mandatory fields are not displayed. This is very irritating.
I have found a workaround for this: if the type selection field is defined as repeatable, the problem does not occur. So this bug is obviously kind of specific. I'm not sure, if it only applies to DSpace CRIS or also to basic DSpace, as I had the impression, that it only applied, when I used the submission form to edit items (and not on initial publication).

To sum it up: This bug applies, if
* the document type field in the submission form is not repeatable AND
* there are type bound fields, which are mandatory AND
* the publication is from a different type
Maybe it also matters, if the field definitions in the submission form are on different pages or if the publication is edited with the publication form.

**Comment by abollini** [comment]
solved in the above PR
